OctopusDSC/Tests/cTentacleAgent.Tests.ps1

```diff
@@ -126,8 +126,42 @@ try
                     Mock Get-MachineFromOctopusServer {return $mockMachine}
                     Mock Get-APIResult {return [string]::Empty}
                     Mock Get-TentacleThumbprint { return "ABCDE123456" }
+                    Mock Get-Space { return @{
+                        "Id" = "Spaces-1"
+                    }}
                     Test-TargetResource @desiredConfiguration | Should Be $true
                 }
+
+                It 'Throws an error when space does not exist' {
```

Comment:
Great to see a test for this - would love to see a lot more though.

Would be great to see some similar to the Server "invocation tests" if possible